generic_config_updater/gu_common.py

```diff
@@ -124,16 +126,17 @@ def validate_config_db_config(self, config_db_as_json):
 
             sy.validate_data_tree()
 
-            for supplemental_yang_validator in supplemental_yang_validators:
-                success, error = supplemental_yang_validator(config_db_as_json)
+            for supplemental_yang_validator, validator_params in \
+                    zip(supplemental_yang_validators, validators_params):
+                success, error = supplemental_yang_validator(*validator_params)
```

Comment:
>supplemental_yang_validator [](http://example.com/codeflow?start=33&length=27)

To make code more readable, you may use lambda in `supplemental_yang_validators`, and no need to `zip` with another array `validators_params`.